js/index.js

```diff
@@ -38,5 +38,86 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+let nav = document.querySelectorAll("a")
+console.log(nav)
+nav[0].textContent = siteContent["nav"]["nav-item-1"]
+nav[1].textContent = siteContent["nav"]["nav-item-2"]
+nav[2].textContent = siteContent["nav"]["nav-item-3"]
+nav[3].textContent = siteContent["nav"]["nav-item-4"]
+nav[4].textContent = siteContent["nav"]["nav-item-5"]
+nav[5].textContent = siteContent["nav"]["nav-item-6"]
+nav[0].style.color = "green"
```

Comment:
Could also change the colors using a function if you want to.

let name = (a, name) => a.style.color = 'green'